src/Client.js

```diff
@@ -810,6 +810,45 @@ class Client extends EventEmitter {
         }
     }
 
+    /**
+     * Get the formatted number of a WhatsApp ID. 
+     * Will return null if the number is not registered on WhatsApp.
+     * @param {string} number Number or ID ("@c.us" will be automatically appended if not specified)
+     * @returns {Promise<string|null>}
+     */
+    async getFormattedId(number) {
```

Comment:
Same as the other comment, maybe a better name is `getFormattedNumber`

Reply:
Agreed